firmware/arduino/chessboard/movementFunctions.ino

```diff
@@ -139,6 +139,13 @@ bool moveDirect(uint8_t startCol, uint8_t startRow, uint8_t endCol, uint8_t endR
   // Turn electromagnet off
   digitalWrite(ELECTROMAGNET, LOW);
 
+  // Print debug info about moveDirect main movement
+  if (DEBUG >= FUNCTION_LEVEL)
+  {
+    Serial.print("Electromagnet State: Off");
+    Serial.println("\n");
```

Comment:
You can just turn the above into a println and add “/n” to the end for fewer print statements